backend/editor/api.py

```diff
@@ -101,7 +101,7 @@ def check_single(id):
     if len(id) == 0:
         raise HTTPException(status_code=404, detail="Entry not found")
     elif len(id) > 1:
-        raise HTTPException(status_code=500, detail="Multiple entries found")
+        raise HTTPException(status_code=409, detail="Multiple entries found")
```

Comment:
I believe that this should still be a 500 error: this error is not user-based and should not happen (if it happens, it means that we have not properly checked the nodes we have added)
```suggestion
        raise HTTPException(status_code=500, detail="Multiple entries found")
```